sympy/concrete/tests/test_products.py

```diff
@@ -382,3 +382,13 @@ def test_rewrite_Sum():
 def test_KroneckerDelta_Product():
     y = Symbol('y')
     assert Product(x*KroneckerDelta(x, y), (x, 0, 1)).doit() == 0
+
+def test_issue_20848():
+    _i = Dummy('_i')
```

Comment:
The underscore in `'_i'` is not needed. `Dummy` automatically appends the underscore for the string representation.

Reply:
Sure will change.
Thanks.